### ISSUE TITLE
Mac installer currently not available

### ISSUE THREAD

**Issue by JonasHelming** [eclipsesource/cdt-cloud#3]
Due to infrastructure issues, we can currently not provided a signed Mac installer.

**Comment by AlexandraBuzila** [comment]
@JonasHelming Just to clarify, this is about providing a .dmg file to download, similar to what Theia Blueprint provides for macOS. Is this correct?

**Comment by JonasHelming** [comment]
Yes, the mac signing was broken, but should work now again. @jfaltermeier should know I guess

**Comment by jfaltermeier** [comment]
Back then the code was committed as a branch on the regular blueprint project. 
Should we wait until it is available here https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint and set up an own Jenkins fpr CDT Cloud Blueprint?
